Kernel/include/scheduler.h

```diff
@@ -0,0 +1,11 @@
+#include <stdint.h>
+
+void sch_init();
+uint64_t sch_switchProcess(uint64_t * currentRSP);
+int sch_loadProcess(uint64_t functionAddress, uint16_t priority,int argq, char * args[]);
+uint8_t sch_getPID();
```

Comment:
Since this is an internal kernel function, it's smart to define exactly whose PID this returns so we're all on the same page. How does "The PID of the currently-running, or last was-running process" sound? (After all it can't be "The caller's PID, since the kernel isn't a process")